File: day05/src/com/oxysa/outputstrem/Demo05.java
package com.oxysa.outputstrem;

import com.oxysa.streamtool.Tool;

import java.io.FileOutputStream;
import java.io.IOException;

/**
 * @author Yummy
 * @mail 873387751@qq.com
 * @date 2020-3-16 20:39
 * 案例：在指定的路径下(例如: d:/abc)创建一个大小为1G的空文件.
 */
public class Demo05 {
    public static void main(String[] args) {
        //1,创建字节输出流对象 关联目的地源文件
        FileOutputStream fos = null;
        try {
            fos = new FileOutputStream("d:/abc.avi");
            //定义字节数组
            byte[] bys = new byte[1024];
            //3,通过循环实现写入数据
            for (int i = 0; i < 1024 * 100; i++) {
                fos.write(bys);
            }
            System.out.println("写入完成！！！1");
        } catch (IOException e) {
            e.printStackTrace();
        } finally {
            Tool.outClose(fos);
        }
    }
}

File: scala_day01/scala/com/oxysa/forexpression/Demo05.scala
package com.oxysa.forexpression

/**
  * @author Yummy
  * @mail 873387751@qq.com
  * @date 2020-3-23 16:04
  *       循环返回值
  */
object Demo05 {
  def main(args: Array[String]): Unit = {
    /*
      说明：for (i <- 1 to 10) yield i * 3
        1，对1 to 10进行遍历
        2，yilid i*3 将每次循环得到 i*3的结果放入到Vector集合中，并返回给res
        3, i 这里是一个代码块，我们可以对i进行数据处理
     */
    val res = for (i <- 1 to 10) yield i * 3
    println(res)
  }
}
